packages/react-native-version-check/src/providers/store.js

```diff
@@ -83,25 +83,48 @@ const MARKETVERSION_STARTTOKEN = 'softwareVersion">';
 const MARKETVERSION_STARTTOKEN_LENGTH = MARKETVERSION_STARTTOKEN.length;
 const MARKETVERSION_ENDTOKEN = '<';
 export function getLatestVersionFromUrl(url, fetchOptions) {
-  return fetch(url, fetchOptions).then(res => res.text()).then(text => {
-    const indexStart = text.indexOf(MARKETVERSION_STARTTOKEN);
-    let latestVersion = null;
-    if (indexStart === -1) {
-      return Promise.reject(text.trim());
-    }
-
-    text = text.substr(indexStart + MARKETVERSION_STARTTOKEN_LENGTH);
-
-    const indexEnd = text.indexOf(MARKETVERSION_ENDTOKEN);
-    if (indexEnd === -1) {
-      return Promise.reject('Parse error.');
-    }
-
-    text = text.substr(0, indexEnd);
-
-    latestVersion = text.trim();
-    return Promise.resolve(latestVersion);
-  });
+  if (Platform.OS === 'ios') {
+    const VersionInfo = getVersionInfo();
+    url = `http://itunes.apple.com/lookup?bundleId=${VersionInfo.getPackageName()}`;
```

Comment:
this url assigning code should be placed [here](https://github.com/kholiavko-roman/react-native-version-check/blob/7651106fee49364b0bd8feb6d9326418eff019b1/packages/react-native-version-check/src/providers/store.js#L42) and [here](https://github.com/kholiavko-roman/react-native-version-check/blob/7651106fee49364b0bd8feb6d9326418eff019b1/packages/react-native-version-check/src/providers/store.js#L74).

Reply:
No, this is not exactly, app market page. 
This url needs only for fetching app info for getting the latest version. If I replace it in that two places for which you indicated, then if new version will be founded, instead of market page will be opened json document with app info.

But I agree then now we have extra call `getStoreUrlAsync` method for iOS, but we dont use this store URL for `getLatestVersionFromUrl` method. I think it would be better change `get` method, and depending on platform call getLatestVersionFromUrl with static url `http://itunes.apple.com/lookup?bundleId=${VersionInfo.getPackageName()}` for ios or use usually case for android.
You can see this solution it in my latest commit.